minor fixes:

icex2 opened the pull request:
- Video.cpp: treating warnings as errors, missing return NULL in video thread
- Makefile: added -ltinfo, this caused the compiler to throw some missing references on linking

Commit:
minor fixes:
- Video.cpp: treating warnings as errors, missing return NULL in video thread
- Makefile: added -ltinfo due to missing references

Commit:
minor fixes:
- Video.cpp: treating warnings as errors, missing return NULL in video thread
- Makefile: added -ltinfo due to missing references

DragonMinded commented on Video.cpp:
```diff
@@ -133,6 +133,8 @@ void *VideoThread( void *state )
         }
     }
+
+	return NULL;
```
Nit: please no tabs, spaces only!

Commit:
fixed whitespaces
added looping for omxplayer to get correct behavior on IIDX double play for video playback

Commit:
Merge branch 'master' of https://github.com/icex2/serialdvdemu

Commit:
added loop argument for omxplayer (disappeared with the merge somehow...)
